### ISSUE TITLE
Nodejs pdf2png.js example - text is not rendered

### ISSUE THREAD

**Issue by arthabus** [mozilla/pdf.js#11677]
I'm following the below example from this repo for nodejs to render a pdf to an image:
https://github.com/mozilla/pdf.js/blob/master/examples/node/pdf2png/pdf2png.js

But it doesn't render any text at all. 

After surfing though other issues here on github, I've added `disableFontFace:false` which partially fixed the problem - it now renders some text while other parts of it are still missing.

I'm seeing a bunch or warnings in server logs like below:

```
Warning: Error during font loading: Font HelveticaLTStd-Bold is not available
Warning: fontRes not available
Warning: TypeError: Cannot read property 'length' of undefined
Warning: Error during font loading: this._map[charCode].charCodeAt is not a function 

```
Is it possible to configure pdf.js to render all text despite of font availability (fallback to default or something)?

Attach (recommended) or Link to PDF file here:
[pdf-example.pdf](https://github.com/mozilla/pdf.js/files/4302207/pdf-example.pdf)

Configuration:
- Web browser and its version: **-** 
- Operating system and its version: **nodejs 8**
- PDF.js version: **2.2.228**
- Is a browser extension: **no**

Steps to reproduce the problem:
1. Use the example code from this repo: 
https://github.com/mozilla/pdf.js/blob/master/examples/node/pdf2png/pdf2png.js
2. add `disableFontFace:false`
3. use this sample pdf to create an image: 
[pdf-example.pdf](https://github.com/mozilla/pdf.js/files/4302206/pdf-example.pdf)
4. check the resulting image.

What is the expected behavior? (add screenshot)
**it renders all the text:**
<img width="565" alt="Screen Shot 2020-03-07 at 18 13 02" src="https://user-images.githubusercontent.com/8145834/76147696-59bf5780-609f-11ea-8356-0f1b5d09ab6d.png">

What went wrong? (add screenshot)
**It doesn't render all the text, producing warnings `Error during font loading` :**
<img width="609" alt="Screen Shot 2020-03-07 at 18 14 43" src="https://user-images.githubusercontent.com/8145834/76147728-8bd0b980-609f-11ea-8c8f-4625aa0fa7a7.png">

**Comment by Snuffleupagus** [comment]
> I'm seeing a bunch or warnings in server logs like below:
> 
> ```
> Warning: Error during font loading: Font HelveticaLTStd-Bold is not available
> Warning: fontRes not available
> Warning: TypeError: Cannot read property 'length' of undefined
> Warning: Error during font loading: this._map[charCode].charCodeAt is not a function 
> ```

This part of the issue should have been fixed by PR #11218; any remaining issues would be a duplicate of issue #4244.

**Comment by arthabus** [comment]
oh that's a bummer.. It doesn't seem to fallback to the default font as per PR #11218 (which would be just great) and the #4244 is from 2014... any chances that there might be a fix/workaround any soon?
